### DISCUSSION TITLE
v19.2.0

### DISCUSSION THREAD

**Discussion by Kyon147** [Kyon147/laravel-shopify#245] [Announcements]
## What's Changed
* Include "host" in @sessionToken output by @michaellehmkuhl in https://github.com/Kyon147/laravel-shopify/pull/194
* Include "shop" in @sessionToken output (to fix POS session issue) by @tom-rice in https://github.com/Kyon147/laravel-shopify/pull/217
* Bump API version to 2023-10 by @npabisz in https://github.com/Kyon147/laravel-shopify/pull/221
* Added Event Listener system by @nahid in https://github.com/Kyon147/laravel-shopify/pull/212

## New Contributors
* @tom-rice made their first contribution in https://github.com/Kyon147/laravel-shopify/pull/217
* @nahid made their first contribution in https://github.com/Kyon147/laravel-shopify/pull/212

**Full Changelog**: https://github.com/Kyon147/laravel-shopify/compare/v19.1.0...v19.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Kyon147/laravel-shopify/releases/tag/v19.2.0'>v19.2.0</a>.</em>

**Comment by nahid** [comment]
Thanks @Kyon147 for released this features. BTW, it's my second contribution

**Reply by Kyon147** [reply]
Oh GitHub just chooses it somehow with the generate release notes.
